_data/import-public.rb

```diff
@@ -35,10 +35,14 @@ def add_files_for_new_team_members
   team_json_data = JSON.parse(team_data)["results"]
 
   team_json_data.each do |team_member_data|
-    team_member_name = team_member_data["name"]
+    if team_member_data["deprecated_name"]
+      team_member_name = team_member_data["deprecated_name"].gsub('.','-')
+    else
+      team_member_name = team_member_data["name"].gsub('.','-')
+    end
     team_member_file_path = File.join(TEAM_DIR, "#{team_member_name}.md")
-
-    unless File.exist?(team_member_file_path)
+    ignored = File.readlines('.gitignore')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.